src/app/core/session/session-service/session-manager.service.ts

```diff
@@ -84,20 +92,38 @@ export class SessionManagerService {
    * Login a offline session without sync.
    * @param user
    */
-  offlineLogin(user: AuthUser) {
+  offlineLogin(user: SessionInfo) {
     return this.initializeUser(user);
   }
 
-  private async initializeUser(user: AuthUser) {
+  private async initializeUser(user: SessionInfo) {
     await this.initializeDatabaseForCurrentUser(user);
-    this.currentUser.next(user);
+    // TODO can we remove this?
+    this.sessionInfo.next(user);
```

Comment:
Do we use it via DI anywhere? Maybe if needed only rarely, we could instead make it part of SessionManagerService only?

Reply:
It's surprisingly still quite a few places where we need this. So I would keep it a separate subject for now.